ziti/cmd/ziti/cmd/lets_encrypt/KeyTypeVar.go

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
It seems weird to me to have a folder under `cmd` in a go project that doesn't actually generate a binary. From this PR, I didn't find a `func main()` so I assume this is a module meant to be included in something that does result in a binary.

Reply:
I followed the pattern already established by `openziti/ziti/ziti/cmd/ziti/cmd/edge_controller`.  I also liked the ability to compartmentalize all this LE stuff in a `lets_encrypt` sub dir.